deploy/docker/events/Dockerfile

```diff
@@ -20,8 +20,10 @@ COPY --from=build --chown=185 /home/jboss/events/target/quarkus-app/*.jar /deplo
 COPY --from=build --chown=185 /home/jboss/events/target/quarkus-app/app/ /deployments/app/
 COPY --from=build --chown=185 /home/jboss/events/target/quarkus-app/quarkus/ /deployments/quarkus/
 
-ENV JAVA_OPTS_APPEND="-XX:-ExitOnOutOfMemoryError -Dquarkus.http.host=0.0.0.0 -Dquarkus.http.port=8000 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:+PrintFlagsFinal"
+ENV JAVA_OPTS_APPEND="-XX:-ExitOnOutOfMemoryError -Dquarkus.http.host=0.0.0.0 -Dquarkus.http.port=8000 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:+PrintFlagsFinal -Dcom.sun.management.jmxremote.port=9091 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
```

Comment:
I think this might be better configured at runtime in app-interface. Disabling remote JMX security may be okay for EE and stage, but probably not for prod. Maybe we move `JAVA_OPTS_APPEND` out of the Dockerfiles and into the deployment environment variables `clowdapp.yml`. Something like:
```yaml
            env:
              - name: JAVA_OPTS_APPEND
                value: "-XX:-ExitOnOutOfMemoryError -Dquarkus.http.host=0.0.0.0 -Dquarkus.http.port=8000 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:+PrintFlagsFinal ${CRYOSTAT_JAVA_OPTS}"
```

`CRYOSTAT_JAVA_OPTS` would be a template parameter inside the `clowdapp.yml` that maybe defaults to empty. In app-interface, we can override this to different values for different environments. This way we can continue work on the Cryostat setup in stage until we're happy with its security, while not blocking any other work from going into prod.

Reply:
Yes, I did not have time to get to this at the end, so in this initial PR the security of this is missing, my intention was to hand over or continue on this with a new PR.

Reply:
Thanks @csviri, I'll handle this part of it.

Reply:
I added a template parameter `CRYOSTAT_JAVA_OPTS` that is set as an environment variable for both deployments. Then I added a shell script to extend `JAVA_OPTS_APPEND` with the value of `CRYOSTAT_JAVA_OPTS`, if defined. This file is sourced automatically by the OpenJDK UBI image. The default value is blank, requiring us to opt-in on individual environments (e.g. stage for now).